packages/editor-ui/src/views/NodeView.vue

```diff
@@ -403,6 +424,13 @@ export default mixins(
 			workflowRunning (): boolean {
 				return this.$store.getters.isActionActive('workflowRunning');
 			},
+			containsTrigger (): boolean {
```

Comment:
if workflow has a trigger node but it's disabled, should we show this or not? Because it's not executable...

Reply:
To me, the current behavior where we enable the button and show an error tooltip that says "Can't be executed, no triggers.." seems alright since the user would have to deliberately disable the trigger, and this would give them a hint to enable it again. But I think it's more of a UX question, so I asked Max/David [in Linear.](https://linear.app/n8n/issue/N8N-4639)

Reply:
To me, the current behavior where we enable the button and show an error tooltip that says "Can't be executed, no triggers.." seems alright since the user would have to deliberately disable the trigger, and this would give them a hint to enable it again. But I think it's more of a UX question, so I asked Max/David in Linear.

Reply:
To me, the current behavior where we enable the button and show an error tooltip that says "Can't be executed, no triggers.." seems alright since the user would have to deliberately disable the trigger, and this would give them a hint to enable it again. But I think it's more of a UX question, so I asked Max/David in [Linear](https://linear.app/n8n/issue/N8N-4639).

Reply:
👍🏽